hoomd/md/methods.py

```diff
@@ -475,21 +499,131 @@ class nph(NPT):
          :math:`W=d N T_0 \tau_P^2`, where :math:`d` is the dimensionality and :math:`N` the number
          of particles.
 
-    :py:class:`nph` is an integration method and must be used with ``mode_standard``.
+    :py:class:`NPH` is an integration method and must be used with ``mode_standard``.
 
     Examples::
 
-        # Triclinic unit cell
-        nph=integrate.nph(group=all, P=2.0, tauP=1.0, couple="none", all=True)
-        # Cubic unit cell
-        nph = integrate.nph(group=all, P=2.0, tauP=1.0)
-        # Relax the box
-        nph = integrate.nph(group=all, P=0, tauP=1.0, gamma=0.1)
+        nph = hoomd.md.methods.NPH(filter=hoomd.filter.All(), tau=1.0, tauS = 1.2, S=2.0)
```

Comment:
```suggestion
        nph = hoomd.md.methods.NPH(filter=hoomd.filter.All(), tauS = 1.2, S=2.0)
```
Remove tau from the other examples as well.